src/metabase/query_analysis.clj

```diff
@@ -106,10 +106,10 @@
      :native     (try
                    (nqa/references-for-native query)
                    (catch Exception e
-                     (log/error e "Error parsing SQL" query)))
-     ;; For now, all model references are resolved transitively to the ultimate field ids.
-     ;; We may want to change to record model references directly rather than resolving them.
-     ;; This would remove the need to invalidate consuming cards when a given model changes.
+                     (log/debug e "Failed to analyze native query" query)))
+       ;; For now, all model references are resolved transitively to the ultimate field ids.
+       ;; We may want to change to record model references directly rather than resolving them.
+       ;; This would remove the need to invalidate consuming cards when a given model changes.
```

Comment:
I don't think the comments should be indented?

Reply:
Yeah looks weird to me too, but if I recall correctly it was cljfmt that did it. Will de-indent and rerun.